docGen/nvd.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"golang.org/x/exp/slices"
```

Comment:
What if bumping Go to 1.21 or 1.22 and using `slices`?

Reply:
Why not.
Created #83 for Go update.

Reply:
@DmitriyLewen I merged this and the  other PRs so you can now use `slices`.

Reply:
Created #84